extra_docs/relations/README.md

```diff
@@ -0,0 +1,163 @@
+# Relations
+All project resources can be related to each other and SDK supports inserting, fetching and attaching/detaching related 
+records.
+Let's create a simple schema which will be used for all further examples:
+
+![Relations schema](https://jexia.github.io/jexia-sdk-js/assets/relations-schema.png) 
+
+There are three datasets and two relations: **posts** has *one-2-many* relation to **comments** and **comments** has 
+*one-2-one* relation to **author** (remember that all management opearations, such as creating datasets, fields and 
+relations can be done with [Web Management Application](https://docs.jexia.com/getting-started/user-management/) only).
+
+To make our datasets not so boring, we can also add a few fields and create appropriate interfaces 
+for our typescript code:
+```typescript
```

Comment:
Is it possible to use JS instead of TS?

Reply:
Yes, but using TS in documentation allows us to show fields and their types... Otherwise we need to make screenshots from webapp dataset fields or do something else.

Reply:
Well, most of our client would be basic JS guys who want to try something. I think it worth to make screenshot. What do you think?

Reply:
I do not understand what do you want here. The code is the same for typescript and javascript, there is no difference at all. These are just interfaces to show what fields we have created in our datasets.